src/util/xml_expr.cpp

```diff
@@ -263,6 +264,15 @@ xmlt xml(
       result.set_attribute("binary", expr.is_true()?"1":"0");
       result.data=expr.is_true()?"TRUE":"FALSE";
     }
+    else if(type.id()==ID_c_bool)
+    {
+      result.name="integer";
+      result.set_attribute("c_type", "_Bool");
```

Comment:
We have to explicitly initialise all parameters passed to the entry function otherwise the following would still fail:
```
void foo(_Bool y)
{
  int x;
  if(y) 
  {  
    x=(int)y;
    assert(x==1);
  }
}
```